types/ckeditor__ckeditor5-core/src/editor/editorconfig.d.ts

```diff
@@ -1,8 +1,7 @@
 import Plugin from "../plugin";
 import { ExportPdfConfig } from "@ckeditor/ckeditor5-export-pdf/src/exportpdf";
 import { ExportWordConfig } from "@ckeditor/ckeditor5-export-word/src/exportword";
-// TODO: import {Alignment} from "@ckeditor/ckeditor5-alignment/src/alignment"
-type Alignment = any;
+import { AlignmentConfig } from "@ckeditor/ckeditor5-alignment/src/alignment";
```

Comment:
```suggestion
import Alignment from "@ckeditor/ckeditor5-alignment/src/alignment";
```
or
```ts
import { Alignment } from "@ckeditor/ckeditor5-alignment";
```

Reply:
@peterblazejewicz This is not right. `Alignment` is a `Plugin` class, and `AlignmentConfig` is an interface that defines the options for the plugin. Check that every other property uses `somethingConfig`. It was my error in `@ckeditor/ckeditor5-core` that it reads `Alignment`. :smile:

Reply:
thx!